drivers/include/net/netdev.h

```diff
@@ -309,6 +309,7 @@ typedef enum {
     NETDEV_CC2538,
     NETDEV_DOSE,
     NETDEV_ENC28J60,
+    NETDEV_ENCX24J600,
```

Comment:
New devices must be added at the end, otherwise L2 addresses of existing devices change

Reply:
Thanks, fixed in 0ca04df